Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -717,7 +735,7 @@ public ResultCode SetNpadJoyAssignmentModeSingle(ServiceCtx context)
         // SetNpadJoyAssignmentModeDual(uint HidControllerId, nn::applet::AppletResourceUserId)
         public ResultCode SetNpadJoyAssignmentModeDual(ServiceCtx context)
         {
-            ControllerId hidControllerId      = (ControllerId)context.RequestData.ReadInt32();
+            PlayerIndex  hidControllerId      = HidUtils.GetIndexFromNpadIdType((NpadIdType)context.RequestData.ReadInt32());
```

Comment:
```suggestion
            PlayerIndex hidControllerId      = HidUtils.GetIndexFromNpadIdType((NpadIdType)context.RequestData.ReadInt32());
```
Don’t forget to remove the space in the line above too